### PR TITLE
Restructuring and a few other improvements

### PR DESCRIPTION
**Pull request by jlevy**
This is on top of #158 (which should be merged first to show this better).

Normally this should be split into multiple PRs but trying to get more done tonight, so it may be best to review this commit by commit.

Restructuring:

- [x] Split up and move part of long intro to Basics section
- [x] Move offers questions to offers section
- [x] Tighten intro, move up scope
- [x] Explain why and that "TLDR" readers should look elsewhere
- [ ] Further review and thinking on renaming Evaluating Equity Comp section

Content:

- [x] Acceleration def confusingly split
- [x] More clarity about general risk vs early stage startup risk
- [x] More clarity on company, stock corporation defs

Production:

- [ ] No incidental definitions boldfacing
- [x] Remove section links when mouseovers would do

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,132 +2,96 @@
...
-[on GitHub](https://github.com/jlevy/og-equity-compensation). We thank
-[everyone who has helped](#credits) write, review, edit, and improve this work.
 
-### Should you use this Guide?
+In contrast, the goals of this Guide are to be **practical** (including lots of concrete
+details, advice, and pitfalls to avoid), **clear** (brief yet precise), and **thoughtful**
+(with context and multiple expert perspectives, including divergent opinion on
+controversial topics).
+
+Perhaps most importantly, Holloway Guides are **imperfect but improving**: It will continue
```
Comment (RacheltheEditor) [inline README.md]: I struggle with the word "imperfect." "Holloway Guides are **improving**" is a positive framing of the enterprise, without the fear prompted by the word "imperfect." "Improving" _implies_ that it is imperfect, so I think it's enough on its own. But the word itself, "imperfect," could build mistrust in the material that _is_ present (not what we want!), while "improving" just lets everyone know that there is material that we're missing and hoping to add, maybe even with your help.

File: README.md [hunk 2]
```diff
@@ -151,6 +115,11 @@ side of the table, either!
 Negotiating with candidates and fielding questions from candidates and employees requires
 understanding the same complex technicalities of equity compensation well.
 
+That said, this topic is **not simple** and we ask that readers be willing to invest time to
+get through a lot of confusing detail.
+If you’re in a hurry, or you don’t care to learn the details, this Guide may not be for
```
Comment (RacheltheEditor) [inline README.md]: Like I said in INTERNAL, I'm not happy telling readers what they want or what we think we know about them. They could come for a quick answer, read a little, and think, "Oh, I do want details, this is interesting or helpful." We have to give them the chance to surprise themselves. Selecting readers is not fair or safe, especially at this stage. This line also sounds like a #humblebrag. Cut. 

"This topic is not simple, and to understand how to make the best decisions for your personal situation, you have to slough through a lot of detail. But we're here to guide you through, and remember that you can ask questions along the way."

File: README.md [hunk 3]
```diff
@@ -151,6 +115,11 @@ side of the table, either!
...
+you. [Seek advice](#seeking-professional-advice).
+
 ### A note on fairness
```
Comment (RacheltheEditor) [inline README.md]: Any thoughts on my thoughts on this section from INTERNAL?

File: README.md [hunk 4]
```diff
@@ -319,6 +265,63 @@ have an understanding of the topic yourself, and know what questions to ask.
 
 ## Equity Compensation Basics
 
+### History and significance
```
Comment (RacheltheEditor) [inline README.md]: 👍

File: README.md [hunk 5]
```diff
@@ -319,6 +265,63 @@ have an understanding of the topic yourself, and know what questions to ask.
...
+[162(m)](https://www.treasury.gov/resource-center/tax-policy/tax-analysis/Documents/Firms-Exceeding-162m.pdf).
+Unfortunately, the legislation backfired, as it created a loophole that made
+performance-based pay—including stock options—fully tax deductible, thereby creating a
+dramatic incentive to pay executives through stock options ([Balsam](https://www.epi.org/publication/taxes-executive-compensation/)).
+From 1970–79, the average compensation for a CEO of one of the 50 largest firms in the
+United States was **$1.2M**, of which **11.2%** was from stock options.
+By 2000–05, the same numbers had risen to **$9.2M** and **37%**, respectively ([Frydman & Jenter](http://www.nber.org/papers/w16585.pdf),
+Fig. 2).
+
+### Growth and risk
```
Comment (RacheltheEditor) [inline README.md]: 👍😄

File: README.md [hunk 6]
```diff
@@ -943,21 +960,25 @@ There are some key subtleties you’re likely to come across in the way
...
-  [table](https://www.investopedia.com/terms/c/capitalization-table.asp) (often a spreadsheet or
-  other official record) that [records](https://www.cooleygo.com/what-is-a-cap-table/) the
-  ownership stakes, including number and class of shares, of all shareholders in the
-  company. It is updated as stock is granted to new shareholders.
+
+🔹 If you’re trying to figure out the likely percentage a number of shares will be worth
+in the future, it’s best to know the number of shares that are fully diluted.
+
+∑ Even the fully diluted number may not take into account outstanding convertible
+securities (like convertible notes) that are *waiting* to be converted into stock at a
```
Comment (RacheltheEditor) [inline README.md]: link for "convertible securities" def

File: README.md [hunk 7]
```diff
@@ -434,60 +438,67 @@ issuing stock is how companies often choose to do this.
...
+their stock is worth.
+
+🄳 Most smaller companies, including all startups, are
+[**private companies**](https://en.wikipedia.org/wiki/Privately_held_company) with owners who
+control how those companies operate.
+Unlike a public company, where anyone is able to buy and sell stock, owners of a private
+company control who is able to buy and sell their stock.
+There may be few or no transactions, or they may not be publicly known.
+
+🚧 What are public exchanges and how is stock bought and sold in practice?
```
[the remark this answers is not in the document]
Comment (jlevy) [reply README.md]: Yeah good point. Fixed.

File: README.md [hunk 8]
```diff
@@ -433,60 +440,67 @@ issuing stock is how companies often choose to do this.
...
+  Some companies now manage their ownership through online services called *ownership
+  management platforms*, such as [Carta](https://carta.com/). If the company you work for uses
+  an ownership management platform, you will be able to view your stock certificates and
+  stock values online.
+  Younger companies may also choose to keep their stock *uncertificated*, which means your
+  sole evidence of ownership is your contracts with the company, and your spot on the
+  company’s [capitalization table](https://en.wikipedia.org/wiki/Capitalization_table), without
+  having a separate certificate for it.
+
+🄳 [**Outstanding shares**](http://www.investopedia.com/terms/o/outstandingshares.asp) refer to
```
Comment (RacheltheEditor) [inline README.md]: Outstanding shares and percentage ownership still feel right to me in "Counting shares," or perhaps those sections should be combined. Confusing why they're separated by companies and startups info...

File: README.md [hunk 9]
```diff
@@ -1055,24 +1076,27 @@ when the stock is fully owned—still, this is actually one of the least common
...
-  The repurchase right lapses over the service-based vesting period, which is what is meant
-  in this case by the stock “vesting.”
-- ☝️ Restricted stock awards are
-  [not the same thing](https://www.fool.com/knowledge-center/the-difference-between-a-restricted-stock-unit-res.aspx)
-  as [restricted stock units](#rsus).
-- Typically, stock awards are limited to executives or very early hires, since once the
-  value of the shares increases, the tax burden of receiving them (without paying the
-  company for their value) can be too great for most people.
-  Usually, instead of restricted stock, an employee will get stock options.
+🄳 A **restricted stock award** is when a company grants someone stock as a form of
```
Comment (RacheltheEditor) [inline README.md]: See INTERNAL notes on this definition. (TLDR The first line doesn't tell us anything specific abt restricted stock awards, it seems to be just a description of stock equity in general. What does "additional" mean in this context? Additional to what?)

File: README.md [hunk 10]
```diff
@@ -1055,24 +1076,27 @@ when the stock is fully owned—still, this is actually one of the least common
...
+compensation. The stock awarded has additional conditions on it, including a vesting
+schedule, so is called **restricted stock**. Restricted stock awards may also be called
+simply **stock awards** or **stock grants**.
+
+∑ What *restricted* means here is actually
+[complex](https://github.com/jlevy/og-equity-compensation/issues/24). It refers to the fact
```
Comment (RacheltheEditor) [inline README.md]: So the initial description says this is the simplest form of equity, and now it's "actually complex." Again, considering that this description is very technical and uses a lot of terminology that we don't explain until we get to stock options, consider moving to "Less common types of equity."

File: README.md [hunk 11]
```diff
@@ -20,63 +20,77 @@ If you talk to enough employees and hiring managers, you’ll hear stories of ho
...
-
-You shouldn’t need a law degree or MBA to understand how you are paid.
-You shouldn’t have to throw blind trust at a lawyer or tax professional.
-And you shouldn’t have to fail before you succeed.
+this Guide. Each of us are motivated by a single purpose:
+To help the reader navigate a confusing topic.
+The Holloway Guide is based on an earlier edition that has received significant discussion
+and feedback [on Hacker News](https://news.ycombinator.com/item?id=10880726) and
+[on GitHub](https://github.com/jlevy/og-equity-compensation). We thank
+[everyone who has helped](#credits) write, review, edit, and improve this work.
```
[the remark this answers is not in the document]
Comment (jlevy) [reply README.md]: Fixed.

File: README.md [hunk 12]
```diff
@@ -1214,66 +1235,70 @@ price. Options are only exercisable for a fixed period of time, until they **exp
...
+  can help grow the value of a company substantially—often having taken a lower salary in
+  exchange for equity—but end up with no ownership because they’re unable or unwilling to
+  stay for the several years [typically needed](#ipos) before an IPO or sale.
+- On the other side, a few companies and investors
+  [stand by the existing system](https://a16z.com/2016/06/23/options-timing/), arguing that it
+  is better to incentivize people not to leave a company, or that long windows effectively
+  transfer wealth from employees who commit long-term to those who leave.
+- As a side note, there is also an argument that it’s a legal requirement of ISOs to have a
+  90-day exercise window.
+  This is technically [true but misleading](https://news.ycombinator.com/item?id=9254299), as
```
Comment (RacheltheEditor) [inline README.md]: Huh. Where did my edits to this section go?

File: README.md [hunk 13]
```diff
@@ -1214,66 +1235,70 @@ price. Options are only exercisable for a fixed period of time, until they **exp
...
+  it’s possible still to give employees the ability lawfully by changing the nature of the
+  options (as illustrated [many companies](https://github.com/holman/extended-exercise-windows)
+  companies now choosing to do).
+- On balance, it seems many people
+  [🔑didn’t understand](https://medium.com/@ben_mathes/90-days-and-my-six-figure-mistake-a495f4a188e2)
```
Comment (RacheltheEditor) [inline README.md]: And here?

File: README.md [hunk 14]
```diff
@@ -1479,31 +1503,34 @@ less than a year from purchase, but profits from these sales are taxed as ordina
...
-- ❗ AMT is relevant to you if you’re reading this.
-  It’s important to understand because exercising ISOs can trigger AMT. In some cases a *lot*
-  of AMT, *even when you haven’t sold the stock* and have no money to pay.
-  We discuss this [later](#taxes-on-stock-and-options).
+
+🄳 **Long-term capital gains tax** is a tax on the sale of assets held longer than a year.
+Long-term capital gains tax is often lower than ordinary income tax.
+Many investors hold assets for longer than a year in order to qualify for the lesser tax
+burden of long-term capital gains.
+🄳 **Alternative Minimum Tax (AMT)** is an
```
Comment (RacheltheEditor) [inline README.md]: Sill consider whether you'd like to combine this with the details on AMT risks in "A bit on how all this fits together" section... (See INTERNAL notes in that section.)

File: README.md [hunk 15]
```diff
@@ -1963,7 +1990,7 @@ to taxes on equity compensation:
   and include a self-addressed, stamped envelope for the IRS to send you a return receipt.
   (Some people are so concerned about this they even ask a friend to go with them to the
   post office as a witness!)
-- ❗ Watch out for [the AMT trap](#the-amt-trap) we’ve already discussed.
+- ❗ Watch out for the AMT trap we’ve already discussed.
```
Comment (RacheltheEditor) [inline README.md]: These tax dangers can and should also be doled out to their relevant sections. See INTERNAL.

File: README.md [hunk 16]
```diff
@@ -2127,71 +2154,6 @@ all together when trying to assign value to your equity:
...
-There are several other
-[resources](https://blog.wealthfront.com/stock-options-14-crucial-questions/) with
-[more questions](http://www.inc.com/atish-davda/5-questions-you-should-ask-before-taking-a-start-up-job-offer.html)
-like this to consider.
-
-🚧 Summarize the best items in the links above.
-
 ### Stock option scenarios
 
 Stock options are an exercise in deciding when to exercise and when to sell, if you can.
```
Comment (RacheltheEditor) [inline README.md]: Rephrase. Cutesy, could confuse ("an exercise in deciding when to exercise"). "...and figuring out if and when you can sell."

File: README.md [hunk 17]
```diff
@@ -2395,12 +2358,78 @@ startup:
...
-  small and growing company, because your role and the roles of others may change a lot, and
-  quickly. It’s more important that you respect the founders and leaders of the company.
+- **Title**: Negotiating title and exact details of your role early on may not matter as much
+  in a small and growing company, because your role and the roles of others may change a
+  lot, and quickly.
+  It’s more important that you respect the founders and leaders of the company.
   It’s more important that you feel
   [you are respected](https://blog.shrm.org/blog/respect-and-trust-top-the-list-of-most-important-employee-job-satisfaction).
 
+### Questions candidates can ask
```
Comment (RacheltheEditor) [inline README.md]: 👍

File: README.md [hunk 18]
```diff
@@ -25,11 +25,12 @@ are best understood by tax attorneys, corporate lawyers, and other professionals
 Decisions related to [negotiating an offer](#offers-and-negotiations) and
 [exercising stock options](#stock-option-scenarios), in particular, can have **major financial
 consequences**. Because the value of employee equity is determined by the fate of the
-company, when a company does not succeed, an employee’s equity may be
-[worth nothing](#what-is-private-stock-worth), and the costs of exercising may not be
-recouped. Even when a company does succeed, an employee may suffer
-[catastrophic tax pitfalls](#tax-dangers) because they didn’t anticipate the tax consequences
-of their decisions.
+company, an employee’s equity may be [illiquid](#what-is-private-stock-worth) for a long time
```
Comment (RacheltheEditor) [inline README.md]: Probably worth defining "illiquid" in "Vesting and liquidity" or in "Can you sell private stock?" We use the term once (other than here!), in "Taxes on RSUs" and the meaning may not be clear to everyone. Or even here, "an employee's equity may be illiquid--that is, unable to be sold for cash--for a long time..."

File: README.md [hunk 19]
```diff
@@ -25,11 +25,12 @@ are best understood by tax attorneys, corporate lawyers, and other professionals
...
+or ultimately [worth nothing](#growth-and-risk), and taxes and the costs of exercise, if they
+apply, may not be recouped.
+Even when a company is doing well, an employee may suffer
+[catastrophic tax pitfalls](#tax-dangers) because they didn’t fully anticipate the tax
```
Comment (RacheltheEditor) [inline README.md]: Elide "fully" (just for economy of words, that word doesn't add anything to the meaning of "anticipate" IMO). 

Removing "consequences of their decisions" connects the consequences directly to a company doing well, e.g. that there will always be tax consequences if a company does well. That could be fine, just want to make sure that's the meaning you want!

File: README.md [hunk 20]
```diff
@@ -401,17 +401,16 @@ trade. In the United States,
...
 specific state.
 In practice, people often use the word *company* to mean *corporation*.
 
-🄳 A [**C corporation (or C corp)**](https://en.wikipedia.org/wiki/C_corporation) is one type of
-corporation in the United States.
-Specifically, it’s a kind of
+🄳 A
 [**stock corporation (or joint-stock company)**](https://en.wikipedia.org/wiki/Joint-stock_company)
-with certain federal
+is a corporation where ownership is managed using stock.
```
Comment (RacheltheEditor) [inline README.md]: Nice!

**Comment by jlevy** [comment]
OK. Going to merge this now as the restructuring is needed, it seems good enough, and we can do specific follow-on PRs for other updates (intro and copy edits separately).

File: README.md [hunk 21]
```diff
@@ -320,6 +275,63 @@ have an understanding of the topic yourself, and know what questions to ask.
...
+Cash-poor startups persuade early employees to take pay cuts and join their team by
+offering [meaningful ownerships stakes](#typical-employee-equity-levels), catering to hopes
+that the company will one day grow large enough to [go public](#ipos) or
+[be sold](#sales-and-liquidity) for an ample sum.
+More mature but still fast-growing companies find offering compensation linked to
+ownership is more attractive than high cash compensation to many candidates.
+
+With the hope for growth, however, also comes **risk**. Fast-growing large companies often
+hit hard times.
+And [startups](#startups-and-growth) routinely fail or yield no returns for investors or
```
Comment (RacheltheEditor) [inline README.md]: Should it be "fail, yielding no returns"?

**Pull request merged**